### ISSUE TITLE
[Bug]: Wrong error message for SINH -- test infra structure, to support Inf.

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#3266]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):e086b4d
- Hardware parameters:Mac M1 ARM 16GB
- OS type: OSX
- Others:
```

### Actual Behavior

[SQL STATEMENT]: SELECT SINH(0.745e5);
[EXPECT RESULT]:

[ACTUAL RESULT]:
The result is not correct because of unexpected reason,Message: For input string: "+Inf"

**Comment by fengttt** [comment]
This is a test infra issue.    Actual value should be 4.34382 × 10^32354 and we correctly set it to +Inf.   The JDBC driver cannot decode +Inf

**Comment by aressu1985** [comment]
if want to fix this issue ,need to customize the jdbc, I suggest punting it to 0.6，
and will need to discuss whether it is necessary to customize the own jdbc driver for mo according to mysql jdbc driver

**Comment by fengttt** [comment]
Or the correct answer should be an error -- out of range.

**Comment by daviszhen** [comment]
I will talk with @dengn tomorrow

**Comment by qingxinhome** [comment]
> out of range.

OK， I plan to refer to the processing habits of MySQL. When the calculation result is infinite, an error is reported: out of range

**Comment by aunjgr** [comment]
I searched mysql documentation and found that there is no sinh at all in mysql. Why do we need this kind of test case?

**Comment by qingxinhome** [comment]
The bug has been solved and merged into main and 0.6 branches. Please test

**Comment by aressu1985** [comment]
fixed:ec44102df1ad19c96a6e580bd9811fe5fdc8584c
mysql> SELECT SINH(0.745e5);
ERROR 1690 (HY000): data out of range: data type float64, DOUBLE value is out of range in 'sinh(74500)'
mysql>
